modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderItemServiceImpl.java

```diff
@@ -341,6 +341,25 @@ public int getCommerceOrderItemsQuantity(long commerceOrderId)
 			commerceOrderId);
 	}
 
+	@Override
+	public List<CommerceOrderItem> getSupplierCommerceOrderItems(
+			long customerCommerceOrderItemId, int start, int end)
+		throws PortalException {
+
+		CommerceOrderItem commerceOrderItem =
+			commerceOrderItemLocalService.fetchCommerceOrderItem(
+				customerCommerceOrderItemId);
+
+		if (commerceOrderItem != null) {
```

Comment:
what if commerceOrderItem is null?

Reply:
the value of 'customerCommerceOrderItemId' is passed as parameter in the GET method, if they pass a value that is not present the fetch returns null

Reply:
which should return an empty list, but at this point it makes no sense to make the second query